### PR TITLE
Create 2021-07-09-seminar-minutes-2021-04.md

### PR DESCRIPTION
**Pull request by IronConrad**
is it OK？

### CODE REVIEW

**Comment by IronConrad** [comment]
OK？

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: Bot Friday 2021年第4次活动纪要
+author: conrad
+categories: events
+tags:
+  - minutes
+header:
+  teaser: /assets/2021/seminar-03/group.jpg
```
Comment (lijiarui) [inline docs/_posts/2021-07-09-seminar-minutes-2021-04.md:8]: We need image for this PR :)

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+- 地点：句子互动，北京市 海淀区 中关村智造大街 G 座 PNP 孵化器 + Zoom在线视频会议
+- 轮值主席：
+    1. Chair: Conrad
+    2. Vice-Chair: 李佳芮
+
+## Attendees
+
+1. 鲁玉超, PM of BaiDu IOT Platform **本次分享者**
+2. Corand, 5G消息先锋大队中的一员, **本次活动主席**
+3. 李佳芮， 句子互动创始人, **本次活动副主席**
```
Comment (lijiarui) [inline docs/_posts/2021-07-09-seminar-minutes-2021-04.md:25]: 副主席应该是另外一个同学~ 我是临时替换了一下~

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+- 李佳芮：饭到晚了
+- 陈堰平：期待更多的主题
+- Corand：期待更多主题
+
+## 轮值主席任命仪式
+
+下周轮值：
+
+- 主席: 段清华
+- 副主席: Leo Song
```
Comment (lijiarui) [inline docs/_posts/2021-07-09-seminar-minutes-2021-04.md:126]: 主席是 Leo Song，副主席是段清华

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+![chair]({{ '/assets/2021/seminar-03/chair.jpg' | relative_url }})
```
Comment (lijiarui) [inline docs/_posts/2021-07-09-seminar-minutes-2021-04.md:128]: 我们需要这次活动的图片

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+## 集体合影
+
+![group]({{ '/assets/2021/seminar-03/group.jpg' | relative_url }})
+
+## After Party 🍻
+
+![after-party]({{ '/assets/2021/seminar-03/after-party.jpg' | relative_url }})
+
```
Comment (lijiarui) [inline docs/_posts/2021-07-09-seminar-minutes-2021-04.md:137]: 我们需要这次活动的图片

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+下周轮值：
+
+- 主席: 段清华
+- 副主席: Leo Song
+
+![chair]({{ '/assets/2021/seminar-03/chair.jpg' | relative_url }})
+
+## 集体合影
+
+![group]({{ '/assets/2021/seminar-03/group.jpg' | relative_url }})
```
Comment (lijiarui) [inline docs/_posts/2021-07-09-seminar-minutes-2021-04.md:132]: 我们需要这次活动的图片

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by IronConrad** [comment]
227

**Review by IronConrad — COMMENTED** [review]
fix text

**Comment by lijiarui** [comment]
commit these image https://github.com/wechaty/bot5.ml/pull/228 to this PR.

**Review by IronConrad — COMMENTED** [review]
right photos

File: docs/assets/2021/seminar-04/a [hunk 7]
```diff
@@ -0,0 +1 @@
+
```
Comment (lijiarui) [inline docs/assets/2021/seminar-04/a:1]: What is this file mean?

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
+---
+title: Bot Friday 2021年第4次活动纪要
+author: conrad
```
[the remark this answers is not in the document]
Comment (lijiarui) [reply docs/_posts/2021-07-09-seminar-minutes-2021-04.md:3]: Please add your profile md here.

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+### 视频
+
+<div class="video-container" style="
+    position: relative;
+    padding-bottom:56.25%;
+    padding-top:30px;
+    height:0;
+    overflow:hidden;
+">
+<iframe width="560" height="315" src="https://youtu.be/MuL-MzvqC-k" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (lijiarui) [reply docs/_posts/2021-07-09-seminar-minutes-2021-04.md:44]: ping

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+<div class="video-container" style="
+    position: relative;
+    padding-bottom:56.25%;
+    padding-top:30px;
+    height:0;
+    overflow:hidden;
+">
+  <iframe
+    src='{{ '/assets/js/viewer-js/#/assets/2021/seminar-04/luyuchao.pdf' | relative_url }}'
```
[the remark this answers is not in the document]
Comment (lijiarui) [reply docs/_posts/2021-07-09-seminar-minutes-2021-04.md:82]: ping

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
BTW， please accept my invitation to bot5 org by clicking https://github.com/wechaty

**Review by IronConrad — COMMENTED** [review]
i think fix all

**Review by IronConrad — COMMENTED** [review]
hope

File: docs/_posts/2021-07-09-seminar-minutes-2021-04.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (IronConrad) [reply docs/_posts/2021-07-09-seminar-minutes-2021-04.md:82]: fix

[the remark this answers is not in the document]
Comment (IronConrad) [reply docs/_posts/2021-07-09-seminar-minutes-2021-04.md:82]: upload

**Review by IronConrad — COMMENTED** [review]
last

**Review by IronConrad — COMMENTED** [review]
?

**Review by lijiarui — APPROVED** [review]
LGTM！

**Pull request merged**